index.d.ts

```diff
@@ -1,15 +1,10 @@
-import fastify, {
-  FastifyError,
-  FastifyReply,
-  FastifyRequest,
-  RegisterOptions
-} from "fastify"
+import fastify, { FastifyError, FastifyReply, FastifyRequest, RegisterOptions } from "fastify";
+import { DocumentNode, ExecutionResult, GraphQLSchema, Source } from 'graphql';
 import { IResolvers } from "graphql-tools";
-import { Server, IncomingMessage, ServerResponse } from "http";
+import { IncomingMessage, Server, ServerResponse } from "http";
 import { Http2Server, Http2ServerRequest, Http2ServerResponse } from 'http2';
-import graphql, { GraphQLSchema, GraphQLError, Source, DocumentNode, ExecutionResult } from 'graphql';
 
-declare namespace FastifyGQL {
+declare namespace fastifyGQL {
```

Comment:
Is this change of capitalization important?

Reply:
Yup. It the main function matches the namespace, the two declarations are merged.
This ensures that the plugin can be used as it was but it makes the Options and Plugin declaration outside. While the latter might be useless, the former is important if you want to create factory functions that internally register the plugin.